src/mvs.c

```diff
@@ -72,12 +72,12 @@ LispPTR values(int arg_count, register LispPTR *args) {
     fnhead = (struct fnhead *)Addr68k_from_LADDR(POINTERMASK & SWA_FNHEAD((int)caller->fnheader));
     pc = (ByteCode *)fnhead + (caller->pc);
   }
-#ifdef ISC
+#ifdef RESWAPPEDCODESTREAM
```

Comment:
Same thing with SWAPPED_FN_CHECK, except it would have to be rewritten to take as a parameter the name of the function object (fnhead here, FuncObj above).